libcontainer/cgroups/fs/cpu.go

```diff
@@ -45,15 +83,8 @@ func (s *CpuGroup) Set(path string, cgroup *configs.Cgroup) error {
 			return err
 		}
 	}
-	if cgroup.Resources.CpuRtPeriod != 0 {
-		if err := writeFile(path, "cpu.rt_period_us", strconv.FormatInt(cgroup.Resources.CpuRtPeriod, 10)); err != nil {
-			return err
-		}
-	}
-	if cgroup.Resources.CpuRtRuntime != 0 {
-		if err := writeFile(path, "cpu.rt_runtime_us", strconv.FormatInt(cgroup.Resources.CpuRtRuntime, 10)); err != nil {
-			return err
-		}
+	if err := s.SetRtSched(path, cgroup); err != nil {
```

Comment:
Is this meant to be here? In the comment for `ApplyDir` (which I think shouldn't be a new function) you say that:

> ```
> // We should set the real-Time group scheduling settings before moving
> // in the process because if the process is already in SCHED_RR mode
> // adding it will fail.
> ```

Surely setting it afterward will cause it to fail? If not, can you please update the comment to explain when it's okay to use `SetRtSched`?